app/bridge/server.js

```diff
@@ -26,7 +26,10 @@ function createServer(name, version, methods) {
     this.enforceContract();
 
     this.ports = [];
+
     this.listen();
+    // the server register itself when it is ready
+    this.register();
```

Comment:
Yes, but they need to indicate they're ready to accept new clients. The process of creating iframes and 
workers is anychronous, right? That's the way I came up with to tell the smuggler that we're ready here. So I may choose another name, like `ready`, that would be more explicit.